client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -50,6 +50,13 @@
                     @click="buildCollectionFromRules">
                     <span v-localize>Build Collection from Rules</span>
                 </b-dropdown-item>
+                <b-dropdown-divider v-if="!showBuildOptions && selectionMatchesQuery" />
+                <b-dropdown-item
+                    v-if="!showBuildOptions && selectionMatchesQuery"
```

Comment:
This condition should probably go into a computed now that is used more than once :)